lecture_2/homework/db/schema.rb

```diff
@@ -10,18 +10,62 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_03_14_150921) do
+ActiveRecord::Schema.define(version: 2019_03_31_185234) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "archers", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "barricades", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "bows", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer "range", default: 30
+  end
+
   create_table "clans", force: :cascade do |t|
-    t.string "name", null: false
+    t.string "name", default: "unnamed", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["name"], name: "index_clans_on_name", unique: true, where: "((name)::text <> 'unnamed'::text)"
+  end
+
+  create_table "hussars", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "katanas", force: :cascade do |t|
```

Comment:
Jak korzystasz z STI to nie ma potrzeby (a wręcz jest niewskazane) tworzyć tabele dla modeli. STI polega na tym, że wszystkie bronie będą trzymane w tabeli `weapons`

Reply:
Dobra teraz to łapie, własnie tak się zastanawiałem bo w różnym miejscach jak czytalem i patrzyłem na schemy to faktycznie przy sti nie ma takich tabel, ale nie rozumiałem czemu, także teraz to robi sens dziekuję!

Reply:
Jak rozwiniesz skrót STI to "Single Table Inheritance" to trochę sugeruje ile tych tabel powinno być 😃